observations/obs_converters/AIRS/L1_AMSUA_to_netcdf.f90

```diff
@@ -0,0 +1,157 @@
+! DART software - Copyright UCAR. This open source software is provided
+! by UCAR, "as is", without charge, subject to all terms of use at
+! http://www.image.ucar.edu/DAReS/DART/DART_download
+
+program L1_AMSUA_to_netcdf
```

Comment:
This program is not finished. At some point, if someone wanted to take a run at making a converter that reads HDF-EOS and only converts what is needed for the DART observation - here is the place to start.

There is an error_handler() EXIT right at the beginning with a warning message.

Reply:
@timhoar I've put in an issue (https://github.com/NCAR/DART/issues/105) to finish the L1_AMSUA_to_netcdf program.  I probably missing the subtleties of the various AMSUA issues.